js_test.js

```diff
@@ -0,0 +1 @@
+console.log("foo")
```

Comment:
Definition for rule 'keyword-spacing' was not found<br>Unexpected console statement.